.github/workflows/linter.yml

```diff
@@ -1,56 +1,33 @@
----
-###########################
-###########################
-## Linter GitHub Actions ##
-###########################
-###########################
-name: Lint Code Base
```

Comment:
as mentioned on slack, for hm-diag and hm-config where we have Dockerfiles etc we should move the python linting out into the ruff/poetry side in a separate github action, but leave the dockerfile and other linting

just noticed we have a separate hadolinter and then duplicating in here as well...maybe we just remove that hadolint and just use the shell/hadolint from this github super linter?

not sure what you think?

Reply:
also noticed in hm-config we are running bandit but not in hm-diag or hm-pyhelper. Maybe we should add?

Reply:
Agreed. Fixed!